index.js

```diff
@@ -9,13 +9,27 @@ exports.prepare = function (extensions, filepath, cwd, nothrow) {
   var attempts = [];
   var err;
   var onlyErrors = false;
-  var ext = extension(filepath);
-  if (Object.keys(require.extensions).indexOf(ext) !== -1) {
+  var exts = extension(filepath);
+
+  var supportsExtension = Object.keys(require.extensions).indexOf(exts[0]) !== -1;
+
+  if (supportsExtension) {
     return true;
```

Comment:
This early return is not in the master branch. See the configuration for `babel.js` [here](https://github.com/tkellen/js-interpret/blob/master/index.js#L2-L11). That's called [here](https://github.com/tkellen/js-rechoir/blob/master/index.js#L28), which invokes [this](https://github.com/tkellen/js-rechoir/blob/master/lib/register.js#L4).

This early exit prevents any of that from happening and it also prevents exposing any errors with registering the module loader.